### PR TITLE
Issue 1118 GitHub actions

### PR DESCRIPTION
**Pull request by akaiap**
This pull request will allow testing (previously Travis /CI) to be shown through Github Actions instead.

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        node-version: [18.x, 20.x, 22.x]
+
+    steps:
+    - uses: actions/checkout@v4
+    # install system dependencies needed by the 'canvas' package
+    - name: Install dependencies for canvas
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y libcairo2-dev libpango1.0-dev 
+        libjpeg-dev libgif-dev librsvg2-dev
```
Comment (dondi) [inline .github/workflows/node.js.yml:28]: I think this should stay as one line; the error hints that GitHub’s environment is trying to perform the `libjpeg-dev` line _as a command_, and that would be due to its being on another line

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    branches: [ "master" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x, 22.x]
```
Comment (dondi) [inline .github/workflows/node.js.yml:19]: Per our meeting, to simplify things for now, let’s reduce this to just one version until things are more settled

**Review by dondi — COMMENTED** [review]
Looking good—we fixed the merge target and that eliminated all of the extra diffs; plus a couple of comments, particularly the one with the line break which may fix the (current) error

**Comment by akaiap** [comment]
The branch has now passed all the checks and CI testing has been successfully integrated into GitHub Actions!

**Review by dondi — APPROVED** [review]
LGTM

**Pull request merged**